web/templates/compose.hbs

```diff
@@ -90,9 +90,9 @@
                                 </a>
                                 <span id="compose-limit-indicator"></span>
                                 <div class="message-send-controls">
-                                    <button type="submit" id="compose-send-button" class="button small send_message compose-submit-button animated-purple-button">
+                                    <button type="submit" id="compose-send-button" class="send_message compose-submit-button compose-send-or-save-button" aria-label="{{t 'Send' }}">
                                         <img class="loader" alt="" src="" />
-                                        <span>{{t 'Send' }}</span>
+                                        <i class="zulip-icon zulip-icon-send"></i>
                                     </button>
                                     <button class="animated-purple-button message-control-button" id="send_later" tabindex=0 type="button" data-tippy-content="{{t 'Send later' }}">
```

Comment:
This should be a follow-up, but the `message-control-button` class is probably a bad name It's a name overlap with the "message controls buttons" that hover in the message feed in `web/templates/message_controls.hbs`. I think `send-control-button` would be better given how this is used?

Reply:
Yes, `send-control-button` makes a lot more sense to me. And that can probably be cleaned up at the same time as the `compose_control_button` class on the Drafts button, as that's hanging around only for the sake of its Tippy tooltip.